core/services/aws/cognitoTrigger/lambdas/customMessageTrigger.ts

```diff
@@ -0,0 +1,24 @@
+import { CustomMessageTriggerEvent, Context, Callback } from 'aws-lambda'
+
+function customEmailTemplateLambda(
+  event: CustomMessageTriggerEvent,
+  _: Context,
+  callback: Callback<CustomMessageTriggerEvent>
+): void {
+  if (event.triggerSource === 'CustomMessage_SignUp') {
```

Comment:
maybe a switch with finally since we are running callback irregardless of the if match? not important

Reply:
Instead of if/else now used switch statement.[Commit](https://github.com/CraftsmenLtd/BloodConnect/commit/0f24b3435bb015206b77bc03e56f1d2e666aba91#diff-fc3b67c84ca47fb15f8f69ae70d87040298c1f1ab6e04fe691febd5d877686a1)